baybe/utils/plotting.py

```diff
@@ -0,0 +1,64 @@
+"""Utilities that are useful for plotting."""
+
+import json
+import os
+import sys
+import warnings
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sns
+
+
+def create_plots(data: pd.DataFrame, name: str, **kwargs) -> None:
+    """Create plots from a given data frame and save them as a svg file.
+
+    The plots will be saved in the location in which the file that calls this function
+    is being located. This method is intended to be used for plotting the results of the
+    examples, but can also be used for other plots.
+
+    If the ``SMOKE_TEST`` variable is set, no plots are being created and this method
+    immediately returns.
+
+    Using the ``BAYBE_MULTIVERSION_PLOTS`` environment variable, it is possible to
+    create plots for the light and dark version of the documentation. If this variable
+    is not set, a single file named `{name}_check.svg` is created.
+
+    Note that it is necessary to provide keyword arguments for setting up the exact
+    plot.
+
+    Args:
+        data: The data frame containing the data to be plotted.
+        name: The name of the plot that should be created.
+        **kwargs: Keyword arguments. Used for specifying the plot.
```

Comment:
Keyword arguments passed to ``seaborn.lineplot``.

Reply:
Changed.